datashader/data_libraries/dask.py

```diff
@@ -1,9 +1,12 @@
 from __future__ import absolute_import, division
 
 import dask
+import dask.array as da
 import dask.dataframe as dd
 from collections import OrderedDict
 from dask.base import tokenize, compute
+import numpy as np
+import pandas as pd
```

Comment:
We should start adding isort or some other import linting, but could you move these up with the dask import.